lib/_http_client.js

```diff
@@ -302,6 +303,22 @@ ClientRequest.prototype._finish = function _finish() {
   OutgoingMessage.prototype._finish.call(this);
 };
 
+ClientRequest.prototype.write = function write(chunk, encoding, callback) {
+  if (this.res) {
+    this.emit('error', new ERR_REQUEST_WRITE_AFTER_RESPONSE());
```

Comment:
This will allow the `'error'` event to be emitted multiple times, are there other case where this can happen?

Reply:
The alternative would be worse though (throwing outside) no?

Reply:
If I'm not wrong the original `ECONNRESET` is forwarded to the `ClientRequest` instance.